multivac/gather_job_data.py

```diff
@@ -120,6 +121,15 @@ def gather_data(self):
                 results[job_failure_type] += 1
                 results['total'] += 1
 
+            # Take data from run payloads
+            try:
+                with open(f'{self.workflow_runs_dir}/{job["run_id"]}.json') as \
+                        run_file:
+                    run = json.load(run_file)
+                    branch = run['head_branch']
```

Comment:
It would be useful to filter workflows by branch. We might add it in a later PR. But when we do, this check should happen before we detect failure type, because that's an expensive operation. I propose that we load job.json before job.log.

Reply:
done

Reply:
where? :)

Reply:
Please move this block before line 95 
```
            # Load info about jobs from .log, if there are logs
```

Reply:
For now block `# Take data from GitHub Workflow runs API` starts on line 98, under the platform assigning and under the assigning `job_id` variable. Block `# Load info about jobs from .log, if there are logs` starts on line 108.